src/main/apiClient.ts

```diff
@@ -29,7 +35,12 @@ export type LoginWithCredentialsParams = {
   auth?: AuthOptions
 }
 
-type EmailRequestPasswordResetParams = { email: string; redirectUrl?: string }
+type EmailRequestPasswordResetParams = {
+  email: string
+  redirectUrl?: string
+  loginLink?: string
+  returnToAfterResetPassword?: string
```

Comment:
Can we have both redirect URL params next to each other?

Otherwise, as mentioned above:
returnToAfterResetPassword -> returnToAfterPasswordReset